ssr-express/README.md

```diff
@@ -0,0 +1,76 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Introduction
+
+The amp-ssr-express is an [express](http://expressjs.com/) middleware that is typically used by
+developers who want to create a dynamic AMP website and take advantage of the performance boost
```

Comment:
Why 'dynamic' AMP website? We also can't say typically if there are no users.

Suggestion for the introduction: "amp-ssr-express is an [express](http://expressjs.com/) middleware that optimizes page load times for websites using AMP for their canonical pages. The middleware uses the same [server-side-rendering](../srr) optimizations as the [Google AMP Cache](https://developers.google.com/amp/cache/).

Reply:
done.